azurerm/internal/services/synapse/synapse_workspace_resource.go

```diff
@@ -228,6 +229,12 @@ func resourceSynapseWorkspace() *schema.Resource {
 				Optional: true,
 			},
 
+			"customer_managed_key": {
```

Comment:
The name of this attribute appears to be a bit ambiguous, it should be named something that would be a bit more intuitive as to what data the end user should be expected to put here, maybe something like `customer_managed_key_versionless_id`?

Reply:
Agreed! I will change it.